modules/lib/src/main/resources/assets/js/app/inputtype/ui/text/dialog/LinkModalDialog.ts

```diff
@@ -304,9 +307,95 @@ export class LinkModalDialog
         const checkbox = Checkbox.create().setLabelText(i18n('dialog.link.formitem.openinnewtab')).setInputAlignment(
             InputAlignment.LEFT).build();
 
+        checkbox.hide();
+
+        checkbox.onRendered(() => {
+            const contentSelector = (<ContentComboBox<ContentTreeSelectorItem>>this.getFieldById(
+                'contentId'));
+            const contentSelectorSelectedContent = contentSelector.getSelectedContent();
+
+            const radio = <RadioGroup>this.getFieldById('contentMediaRadio');
+            const radioValue = radio.doGetValue();
+
+            contentSelector.onValueChanged(() => {
+                const contentSelectorSelectedContent = contentSelector.getSelectedContent();
```

Comment:
It feels a bit strange to assign listener of content selector's `valuechanged` event inside `onrendered` listener of a totally different control. It's better to assign this listener right after the content selector is created